packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -304,6 +308,10 @@ export class AttributesController extends ConsumptionBaseController {
             succeededBy: parsedSuccessorParams.succeededBy
         });
 
+        if (predecessor.isComplexAttribute()) {
+            await this.succeedChildrenOfComplexAttribute(successor.id);
+        }
```

Comment:
Resolving; we'd need database transactions to properly handle errors in the attribute controller during creation or update of attributes. This may be implemented in the future.